store/mockstore/unistore/cophandler/mpp.go

```diff
@@ -402,6 +403,10 @@ type ExchangerTunnel struct {
 	ErrCh       chan error
 }
 
+func (tunnel *ExchangerTunnel) debugString() string {
```

Comment:
It is not used. Could you please remove it ~

Reply:
Sometimes I would use it to print log to debug mock tiflash. I think leaving this brief , harmless and sometimes useful interface is not a big matter, right?